metrics-pipeline/metrics-pipeline.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
rename 'metrics-pipeline' to just metrics or metrics-visualization  (rename metrics.py if needed)